ocaml/xenopsd/scripts/swtpm-wrapper

```diff
@@ -132,21 +176,32 @@ def main(argv):
             os.chown(tpm_dir, uid, uid)
```

Comment:
I don't think this is right. If the tpm_dir is owned by the same uid as the swtpm process, then it can change the permissions back to 07xx. If the tpm_dir is owned by root, that can't happen since only the owner can change permissions.